lib/open-sesame.coffee

```diff
@@ -5,8 +5,12 @@ LEADING_SLASH_REGEX = /^\//
 OPEN_COMMAND        = 'open-sesame:open-file-under-cursor'
 
 getPathUnderCursor = ->
-  atom.workspace.getActiveTextEditor()?.getWordUnderCursor
-    wordRegex: PATH_REGEX
+  editor = atom.workspace.getActivePaneItem()
+  text = editor?.getSelectedText()
+  unless text.length > 0
```

Comment:
I think what might work nicely here is

```coffee
editor?.getSelectedText() or editor?.getWordUnderCursor 
  wordRegex: PATH_REGEX
```

No `text` variable is necessary, and neither is a conditional statement.

What do you think?